src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -335,6 +333,12 @@ private static void AddPromptToRequest(Prompt prompt, bool isForceLoginPrompt, W
 
                 if (accountProvider == null)
                 {
+#if !WINDOWS_APP
+                    if (WindowsNativeUtils.IsElevatedUser())
+                    {
+                        throw new MsalClientException(MsalError.AuthenticationFailedWamElevatedProcess, MsalErrorMessage.AuthenticationFailedWamElevatedProcess);
```

Comment:
There are two scenario's when this will be hit:
- App is elevated and workaround was not used - Account Picker crashes.
- App is elevated and workaround was used, but the user manually cancelled the dialog.

Reply:
Hmm, this is not ok. "auth_cancelled" is a very common use case that apps handle. We cannot change this behavior. I think we can only change the error message to say "User cancelled auth or x y z"

Reply:
Changed to keep the same error code, but different message.